packages/mobile/README.md

```diff
@@ -6,46 +6,102 @@ Quiet Mobile is a React Native app for Android and iOS that shares a Node.js [ba
 
 ### Prerequisites
 
-Install `patch` (e.g. via your Linux package manager).
+1. If not on Mac (which comes preinstalled with `patch`), install `patch` (e.g. via your Linux package manager).
 
-In the root directory of `quiet/`, install the monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
+1. In the root directory of `quiet/`, install the monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
 
-```
-npm install
-npm run lerna bootstrap
-```
+    ```bash
+    npm install
+    npm run lerna bootstrap
+    ```
+
+1. On your host, install [adb](https://developer.android.com/studio/command-line/adb) (Android Debug Bridge) to communicate with your Android device.
+
+1. If running on a physical device, enable USB debugging on your device and connect it to your computer via USB. If running on an emulator, start the emulator.
+
+      [React Native: Running on Device](https://reactnative.dev/docs/running-on-device)
+      [Android Developers: Configure Developer Options](https://developer.android.com/studio/debug/dev-options)
+
+1. Add the path to the java binary installed by Android Studio JAVA_HOME to your environment variables. Ensure that you have followed the SDK installation instructions in the [React Native Development Environment](https://reactnative.dev/docs/environment-setup) guide.
+
+    For example, on macOS with Android Studio installed in the default location,
+    Add the following line to your `~/.bash_profile` or `~/.zprofile` file,
+    ```bash
+    export JAVA_HOME="/Applications/Android Studio.app/Contents/jbr/Contents/Home"
+    ```
+
+### Local development
+
+Follow the steps in [React Native Development Environment](https://reactnative.dev/docs/environment-setup) to set up your development environment.
```

Comment:
Why is this necessary?

Reply:
This includes instructions for installing necessary dependencies like `watchman` , Android Studio, Xcode, and setup of those. It was critical for my setup.

Reply:
Ok, I think one thing that confused me was that when I click that link it's talking about Expo and I don't think we use that, but I'm not even really sure what Expo is